docs/JAVA11.md

```diff
@@ -4,18 +4,33 @@ To run the tests using a Java 11 virtual machine:
 
 1. On the host, launch the container with Java11 VM:
 
+
    ```bash
    harry@devbox:~/acceptance-test-harness$ env java_version=11 ./ath-container.sh
    ```
 
-1. In the container shell, set up the vnc and run the tests with Java11 support:
+1. In the container shell, set up the vnc:
 
    ```bash
    ath-user@1803848e337f:~/ath-sources$ eval $(./vnc.sh)
    ```
 
+1. And run the tests with Java11 support:
+
+   Most likely, you will need to add some modules left apart from Java 9, like `JAVA_OPTS="--add-modules java.sql"` used by _pipeline-maven-plugin_ for example. It depends on the Jenkins version and the plugins to test. You can find the most updated documentation on how to run Jenkins on Java 11 at: https://jenkins.io/doc/administration/requirements/jenkins-on-java-11. Basically:
+
+   For Jenkins [2.155, 2.163), add modules, _enable-future-java_ and dependencies from tests:
```

Comment:
I would simply strongly recommend avoiding these versions on Java 11. Only "experts" should try them (or we put this in some addendum, not in the main docs flow IMO).